handler.js

```diff
@@ -237,122 +236,103 @@ module.exports.build_done = (event, context, callback) => {
         target_url: 'https://' + region + '.console.aws.amazon.com/codebuild/home?region=' + region + '#/builds/' + event.build.id + '/view/new',
         context: githubContext,
         description: 'Build ' + buildStatus + '...'
-      }).catch(function(err){
-        console.log(err);
-        context.fail(data);
-      });
+      }).catch(function (err) {
+        console.log(err)
+      })
     }
-  });
+  })
 }
 
-function setGithubAuth(octokit, ssm, params, callback) {
+function setGithubAuth (ssm, params, callback) {
+  console.log('Setting up the Github auth object')
+  var octokit
 
-  if (octokit.hasOwnProperty("auth")) {
-    console.log("Github auth object already set");
-    callback();
-  } else {
-    console.log("Setting up the Github auth object");
-
-    var cred = {
-      type: "basic"
-    };
-
-    ssm.getParameter(params.username, function (err, data) {
-      if (err) callback(err);
-      else {
-        cred.username = data.Parameter.Value;
-        ssm.getParameter(params.accessToken, function (err, data) {
-          if (err) callback(err);
-          else {
-            cred.password = data.Parameter.Value;
-            try {
-              octokit.authenticate(cred);
-            } catch (err) {
-              callback(err);
-            }
-            callback();
-          }
-        });
+  // attempt to get credentials
+  ssm.getParameter(params.accessToken, function (err, data) {
+    if (err) {
+      callback(err)
+    } else {
+      try {
+        octokit = new Octokit({
+          auth: data.Parameter.Value
+        })
+        callback(null, octokit)
+      } catch (err) {
+        callback(err)
       }
-    });
-  }
+    }
+  })
 }
 
-/*
```

Comment:
why remove the docstring?

Reply:
because I wasn't paying attention to what it was and thought it was just commented out code